src/halmos/config.py

```diff
@@ -45,10 +47,39 @@ def arg(
             "short": short,
             "countable": countable,
             "global_default_str": global_default_str,
+            "action": action,
```

Comment:
would it be better to use a different name, e.g., `custom_action`, to avoid potential name clashes? it might be fine as the name `choices` is also used here, but let me know if you have any concerns.

Reply:
not sure I follow, name clash with what? 

I'm ok with `custom_action`, but don't love the "action" nomenclature. If this is about parsing, what about `parser`?

Reply:
`parser` is even more overloaded, ignore that. Either `action` or `custom_action` works IMO, I wouldn't worry about name clashes at that level (I thought this was a concern about shadowing)

Reply:
okay, i will stick with `action`.